src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -337,7 +337,6 @@ private void TestMessageHandler(object sender, TestRunMessageEventArgs e)
                     Output.Warning(ConsoleLogger.AppendPrefix, e.Message);
                     break;
                 case TestMessageLevel.Error:
-                    this.testOutcome = TestOutcome.Failed;
```

Comment:
Why this is removed? This impact behavior. If the adapter/platform send any error message after this change we not failing test run.

Reply:
Will fix this and add an UT. Currently didn't see any UT fail.. Also why is the failure of a run decided at a logger level ?